app/user/repository.py

```diff
@@ -0,0 +1,26 @@
+import abc
+
+from sqlalchemy import select
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from app.db import tables as tb
+
+
+class BaseUserRepository(abc.ABC):
+    def __init__(self, session: AsyncSession) -> None:
+        self.session = session
+
+    @abc.abstractmethod
+    async def findUser(self, id: str) -> str:

```

Comment:
service에서도 메서드는 snake case를 채택했으니, 여기서도 포맷을 통일시켜주면 좋을 것 같아요! (p4)
```suggestion
    async def find_user(self, id: str) -> str:

```

Reply:
전체적으로 camel case to snake case 작업을 진행했습니다!
84c056e77b01e74ea77b230e5bea4efdd46f2e24